<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDisplay</class>
 <widget class="QDialog" name="DialogDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Motor0     Angle:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="angle0">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="speed0">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="speed1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="angle1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Motor1     Angle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="angle2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Motor2     Angle:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="speed2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Motor3     Angle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="speed3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>190</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="angle3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="angle4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Motor4     Angle:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="speed4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>250</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="speed9">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>630</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="angle9">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>630</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>390</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Motor5     Angle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>630</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Motor7     Angle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Motor9     Angle:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="angle6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>450</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Motor6     Angle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>570</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="angle8">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>570</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="angle7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>510</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="speed8">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>570</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>510</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="angle5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Motor8     Angle:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="speed5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>390</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="speed6">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>450</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>450</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="speed7">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>510</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>6</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
